kmk/kmk_keyboard.py

```diff
@@ -7,7 +7,6 @@
 import gc
 
 from kmk import led, rgb
-from kmk.ble import BLEHID
```

Comment:
I believe that this is a mistake. This needs reverted in order to cleanly merge.